objectivec/GPBDescriptor.m

```diff
@@ -851,31 +853,33 @@ - (BOOL)getValue:(int32_t *)outValue forEnumName:(NSString *)name {
   nameAsCStr += prefixLen;
 
   if (nameOffsets_ == NULL) [self calcValueNameOffsets];
-
-  // Find it.
-  for (uint32_t i = 0; i < valueCount_; ++i) {
-    const char *valueName = valueNames_ + nameOffsets_[i];
-    if (strcmp(nameAsCStr, valueName) == 0) {
-      if (outValue) {
-        *outValue = values_[i];
+  if (nameOffsets_) {
```

Comment:
Lets go ahead an used the early return style in all of these like you did for `getEnumNameForIndex:` and `getEnumTextFormatNameForIndex:` that way is will hopefully be more clear going forward how to handle a failure with calc.